Orange/widgets/visualize/widgetutils/tree/rules.py

```diff
@@ -88,11 +90,11 @@ class ContinuousRule(Rule):
 
     Examples
     --------
-    x ≤ 30
-    >>> rule = ContinuousRule('age', False, 30, inclusive=True)
+    >>> ContinuousRule('age', False, 30, inclusive=True)
+    age ≤ 30.000
 
-    x > 30
-    >>> rule = ContinuousRule('age', True, 30)
+    >>> ContinuousRule('age', True, 30)
+    age > 30.000
```

Comment:
Sorry to have mislead you, this is not ok. `__repr__()` [should return a Python expression from which the object can be recreated](https://docs.python.org/3/reference/datamodel.html#object.__repr__).  So best use separate repr and str implementations.